CRM/Contribute/Form/ManagePremiums.php

```diff
@@ -55,11 +56,48 @@ public function setDefaultValues() {
       if (isset($tempDefaults['period_type'])) {
         $this->assign('showSubscriptions', TRUE);
       }
+
+      // Convert api3 field names to custom_xx format
```

Comment:
Check out the function `CRM_Core_BAO_CustomField::getShortNameFromLongName()`

Reply:
fixed - thanks!